src/controller_examples/rabbitmq_controller/spec/types.rs

```diff
@@ -131,7 +131,12 @@ impl ResourceView for RabbitmqClusterView {
     proof fn unmarshal_result_determined_by_unmarshal_spec_and_status() {}
 
     open spec fn state_validation(self) -> bool {
-        true
+        &&& self.spec.replicas >= 0
```

Comment:
Can you also update the crd yaml with this validation rule?

Reply:
Done